conf/timeseries-filodb-server-consul.conf

```diff
@@ -5,6 +5,8 @@ filodb {
     max-num-partitions = 5000
     groups-per-shard = 10
     shard-memory-mb = 1500
+    # Number of bytes of offheap mem to allocate to write buffers for all shards.
+    ingestion-buffer-mem-size = 400MB
```

Comment:
How are we isolating by dataset today? Do they have separate shards? I don't think dataset isolation is a good idea, because it makes memory configuration more difficult.

Reply:
Shards are under datasets in the hierarchy if you look at `TimeSeriesMemStore`.  Thus we isolate by both dataset and shard.
I think isolation is important.  Today all memory is divided by both dataset and shard.  It becomes very difficult to reason about SLAs if raw data is mixed with downsampled stuff for example.